test/test_proxymanager.py

```diff
@@ -5,12 +5,12 @@
 
 class TestProxyManager(object):
     def test_proxy_headers(self):
-        url = 'http://pypi.python.org/test'
+        url = 'https://pypi.org/project/urllib3/'
```

Comment:
Sorry for the nit but could you turn this back to `http`? I want to maintain the same code path being tested. :)

Reply:
No problem. Done in the latest version.